docs/source/reference/configuration.md

```diff
@@ -71,6 +72,7 @@ These variables are optional that can be specified in the design configuration f
 
 | Variable | Description |
 |-|-|
+| `MODULES_SPEF_FILES` | Specifies min, nom, max spef files for modules(s). Variable should be provided as a json/tcl list or a space delimited tcl string. Note that a module name is provided not an instance name. A module may have multiple instances. Each module must have 3 files for each corner. For example: `module1 min1 nom1 max1 module2 min2 nom2 max2`. <br> (Default: NONE) |
```

Comment:
Would it not be more consistent to call this variable `EXTRA_SPEF`?

Also, can you add the note about using nominal multiple times if the other corners do not exist?

Reply:
Sure